### PR TITLE
feat(discoverv2): tracing-cleanup

### PR DESCRIPTION
**Pull request by doralchan**
Screen Shot 2019-12-02 at 6 09 38 PM

### CODE REVIEW

**Review by dashed — APPROVED** [review]
Looks good to me 👍

File: src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx [hunk 1]
```diff
@@ -937,12 +925,6 @@ const getTogglerTheme = ({
     background: ${buttonTheme.background};
     border: 1px solid ${buttonTheme.border};
     color: ${buttonTheme.color};
-
-    &:hover {
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx]: @doralchan is this intentional?

File: src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx [hunk 2]
```diff
@@ -95,7 +95,7 @@ const Row = ({
       <td className="key">{title}</td>
       <td className="value">
         <pre className="val " style={{backgroundColor: '#F0ECF3'}}>
-          <span className="val-string">{children}</span>
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx]: @doralchan just a heads up, the removal of this classname will affect the font colour of the value field (from purple to black)

Screen Shot 2019-12-03 at 4 42 17 AM

Previously: 

Screen Shot 2019-12-03 at 4 43 30 AM

File: src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx [hunk 3]
```diff
@@ -551,20 +543,14 @@ const TickLabel = (props: {
...
-
   border-radius: 3px;
   border: 1px solid rgba(0, 0, 0, 0.1);
-
   height: 16px;
-
   line-height: 1;
   vertical-align: middle;
-
   transform: translateY(50%);
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx]: @doralchan This likely needs adjusting now that the duration guide box extends slightly outside the time axis component. 
Screen Shot 2019-12-03 at 5 11 12 AM

Would you be interested in looking into this?

**Review by dashed — APPROVED** [review]
@doralchan feel free to merge this PR. Anything that needs to be addressed can be followed up in another PR.

File: src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx (hunk 2 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx]: Yep

File: src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx (hunk 3 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx]: Whoa - how do I view this?

Comment (dashed) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx]: @doralchan  if you hover your mouse hover the minimap, you can see the duration box guide

Comment (doralchan) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx]: Screen Shot 2019-12-03 at 9 22 06 AM
✔️ Okie doke

File: src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx]: Yeah - its kinda weird to have that appear white on hover

Comment (dashed) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx]: Ok gotcha. Just checking 👍

File: src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx (hunk 2 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx]: I just realized this should be consistent with the other code blocks in general

Comment (doralchan) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx]: Screen Shot 2019-12-03 at 9 31 11 AM

✔️ Fixed

File: src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx [hunk 4]
```diff
@@ -551,20 +543,12 @@ const TickLabel = (props: {
...
-  height: 16px;
-
   line-height: 1;
-  vertical-align: middle;
-
-  transform: translateY(50%);
-
   white-space: nowrap;
 
   ${({alignLeft}) => {
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx]: @doralchan may neeed to remove `translateY(50%)` here as well to address the issue when the duration guide box is displayed on the left-hand side
Screen Shot 2019-12-03 at 12 50 58 PM

Comment (doralchan) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/header.tsx]: Screen Shot 2019-12-03 at 9 56 17 AM

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Pull request merged**